endstate_correction/protocol.py

```diff
@@ -124,11 +127,10 @@ class Results:
     dE_qml_to_mm: np.array = np.array([])
     W_mm_to_qml: np.array = np.array([])
     W_qml_to_mm: np.array = np.array([])
-    endstate_samples_mm_to_qml : np.array = np.array([])
-    endstate_samples_qml_to_mm : np.array = np.array([])
-    switching_traj_mm_to_qml : np.array = np.array([])
-    switching_traj_qml_to_mm : np.array = np.array([])
-   
+    endstate_samples_mm_to_qml: np.array = np.array([])
```

Comment:
I think it might be good to move the reformatting to another PR as the diff is not very helpful in this case.